app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,47 @@
+class MoviesController < ApplicationController
+
+  def index
+    movies = Movie.all
+
+    render(
+      json: movies.as_json(only: [:id, :title, :release_date]),
+      status: :ok
+    )
+  end
+
+  def show
+    movie = Movie.find_by(id: params[:id])
+
+    if movie
+      render json: movie.as_json(only: [:id, :title, :release_date, :overview, :inventory, :available_inventory]),
+      status: :ok
+
+    else
+      render(
+        json: {nothing: true}, status: :not_found
```

Comment:
This will send back the literal JSON `{nothing: true}`, which might not be what you intended here.